tasks/main.yaml

```diff
@@ -7,6 +7,20 @@
       set_fact:
         rsyslog_enabled: "{{ logging_enabled|d(true) }}"
 
+    - name: Set rsyslog_use_roles
+      set_fact:
+        rsyslog_use_roles: "{{ logging_use_roles|d(false) }}"
+
+    - name: Set rsyslog_purge_original_conf
+      set_fact:
+        rsyslog_purge_original_conf: "{{ logging_purge_confs|d(true) }}"
+
+    - name: Set ansible_python_interpreter if python3 is used
```

Comment:
Is it necessary to do this here?  Seems like something that should be done outside of this role.

Reply:
Is `ansible_python_interpreter` used only by this role?

Reply:
Thanks @richm for spotting this. `ansible_python_interpreter` must be set by the environment, (outside roles/playbooks/tasks) otherwise nothing works, sio it really does not belong there. Ansible 2.8 and later autodetects it on RHEL8.